proposals/multi-value/Overview.md

```diff
@@ -0,0 +1,196 @@
+# Multi-value Extension
```

Comment:
@binji I only realized now that you added the `proposals/` dir too. Was that intentional?

Reply:
It was; I thought we decided to do this so we could easily see that you could easily see which proposals had been merged into the spec. That said, it was a long time ago and I could be misremembering.

Reply:
At least it matches my memory as well. :)